chevah_build

```diff
@@ -933,7 +932,6 @@ command_test() {
             # available, but they need to be requested for the tests to pass.
             export LC_ALL=en_US.UTF-8
             execute $LOCAL_PYTHON_BINARY ${SCANDIR_FOLDER}/test/run_tests.py
-            export LC_ALL=C
```

Comment:
why this change?
The idea is that all tests should run with `LC_ALL=C`... in this way, Python does not trigger the implicit Unicode conversion.

We run scandir tests with non LC_ALL=C as some tests are poorly written

Reply:
Sorry, my bad! I will revert this…

Had some issues with testing `scandir` on Solaris 10, so I tried something here, intending to also update `scandir`. But things improved by themselves somehow, so there wasn't any need for changes or the update.